python/lang/security/deserialization/avoid-pyyaml-load.yaml

```diff
@@ -25,4 +25,6 @@ rules:
       $YAML = ruamel.yaml.YAML(...)
       ...
       $YAML.load(...)
+  - pattern-not: yaml.load(..., Loader=yaml.CSafeLoader)
+  - pattern-not: yaml.load(..., Loader=yaml.SafeLoader)
```

Comment:
Doesn't this need one more ... after the loader?

Reply:
I think since kwargs can be in any order, it doesn't matter. Hold please...

Reply:
Nope, it needs the `...`.

Reply:
https://semgrep.dev/s/5QKX/?registry=python.lang.security.deserialization.avoid-pyyaml-load